### PR TITLE
Update logging getting started guides to reflect namespace change

### PR DESCRIPTION
**Pull request by satnam6502**
@caesarxuchao @a-robinson

### CODE REVIEW

File: docs/getting-started-guides/logging.md [hunk 1]
```diff
@@ -35,7 +37,8 @@ To help explain how cluster level logging works let’s start off with a synthet
      args: [bash, -c, 
             'for ((i = 0; ; i++)); do echo "$i: $(date)"; sleep 1; done']
 ```
-This pod specification has one container which runs a bash script when the container is born. This script simply writes out the value of a counter and the date once per second and runs indefinitely. Let’s create the pod.
+This pod specification has one container which runs a bash script when the container is born. This script simply writes out the value of a counter and the date once per second and runs indefinitely. Let’s create the pod in the default
+namespace.
```
Comment (caesarxuchao) [inline docs/getting-started-guides/logging.md]: remove the \n?

Comment (caesarxuchao) [inline docs/getting-started-guides/logging.md]: hmm, it's not shown in the rendered text. It's fine then.

**Comment by caesarxuchao** [comment]
LGTM

**Comment by satnam6502** [comment]
How about a merge? @vmarmol 
Thanks.

**Pull request merged**